uiowa/uiowa
# Added partial split in order to adjust the taxonomy term view to exclude topic collection

Resolves #4882

# How to test

1. Check out this branch
2. Pull down the Basic Needs and Support website `blt ds --site=basicneeds.uiowa.edu`
3. Visit the homepage
4. Click on a tag (e.g., "Additional Help")
5. The "Find Resources" page should no longer show up first
